packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -38,7 +36,7 @@ const _useOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedP
   const {
     remoteParticipants = [],
     dominantSpeakers = [],
-    maxRemoteVideoStreams = DEFAULT_MAX_REMOTE_VIDEOSTREAMS,
+    maxRemoteVideoStreams,
```

Comment:
Do we still need to provide a default here?

Reply:
I think we still need a default here is still needed here for stable. I don't think it's bad to keep the default in beta alsol.